shared/components/first.js

```diff
@@ -1,32 +1,24 @@
 import React, {PropTypes, Component} from 'react';
-import {View} from 'react-native';
 import MapLayout from '../containers/mapLayout';
-import HeaderLayout from '../containers/headerLayout';
 
 export default class First extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
I just don't know.
Why should we use constructor method? since we use only super.?